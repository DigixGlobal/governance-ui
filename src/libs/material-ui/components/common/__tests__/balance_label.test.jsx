import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import BalanceLabel from '../balance_label';

describe('<BalanceLabel />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <BalanceLabel
        item={{ color: 'blue' }}
        value="testing"
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
