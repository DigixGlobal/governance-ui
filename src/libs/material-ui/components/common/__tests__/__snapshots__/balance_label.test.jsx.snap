// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BalanceLabel /> renders correctly 1`] = `
<Label
  as="a"
  className="padded"
  color="blue"
>
  testing
  <LabelDetail />
</Label>
`;
